### PR TITLE
test: add a test for failure case

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: test/command/test-run-regression-test.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
+# coding: utf-8
```
Comment (kou) [inline test/command/test-run-regression-test.rb]: Could you remove this?

File: test/command/test-run-regression-test.rb [hunk 2]
```diff
@@ -151,6 +153,55 @@ def test_success
 -  [2, "log message2: 焼肉"]]]
 +  [3, "log message3: 焼肉"]]]
 
+.
+QUIT
+      REQUEST
+    end
+    def test_failure
```
Comment (kou) [inline test/command/test-run-regression-test.rb]: Could you add an empty line before this line?

**Comment by komainu8** [comment]
@kou
Thank you for your comments.
I've fixed them.

**Comment by komainu8** [comment]
@kou Thank you for your comment.
I've fixed it.

**Comment by komainu8** [comment]
@kou Thank you for your comment.
I've fixed it.

**Pull request merged**